LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentBrowserSwitchRequest.kt

```diff
@@ -0,0 +1,22 @@
+package com.braintreepayments.api
+
+import androidx.annotation.RestrictTo
+
+/**
+ * Request parameters associated with a [LocalPaymentPendingRequest.Started]
+ */
+class LocalPaymentBrowserSwitchRequest internal constructor(
+    browserSwitchPendingRequest: BrowserSwitchPendingRequest.Started
```

Comment:
Could we simplify this class by defining the variable in the constructor?

```suggestion
    private val browserSwitchPendingRequest: BrowserSwitchPendingRequest.Started
```

This would remove the need to define `browserSwitchPendingRequest` within the class and the init

Reply:
This is a good call I think it actually warrants a separate PR to align across all the modules that use browser switch. I think we can remove the need for this wrapper class, and just have a `BrowserSwitchPendingRequest.Started` as a `private val` in the `LocalPaymentPendingRequest.Started`. The whole point of this `LocalPaymentBrowserSwitchRequest` was to encapsulate browser switch classes, but if we make that class a private val, it's still encapsulated from the merchant perspective.